pyanaconda/modules/users/users.py

```diff
@@ -173,6 +177,18 @@ def set_ssh_keys_with_task(self, sysroot):
         task = SetSshKeysTask(sysroot=sysroot, ssh_key_data_list=self.ssh_keys)
         return self.publish_task(USERS.namespace, task)
 
+    def configure_root_password_ssh_login_with_task(self, sysroot):
```

Comment:
I guess that this is required because of the initial setup.

Reply:
Yep, IIRC it needs to be able to cherry pick these tasks at the moment. Actually, I think it will have to run the new task as well as it does have the root password spoke in some cases.